### PR TITLE
Improve backlogged episodes count

### PR DESCRIPTION
**Pull request by fernandog**
@OmgImAlexis @duramato

Same:
image

Increase:
image

Decrease:
image

Related to: https://github.com/pymedusa/Medusa/issues/2091

### CODE REVIEW

**Comment by OmgImAlexis** [comment]
This just makes it look tacky. @p0psicles @Thraxis thoughts?

**Comment by fernandog** [comment]
@OmgImAlexis despite UI, code is good?

**Comment by fernandog** [comment]
@p0psicles I already changed what @OmgImAlexis asked. He is ok now

File: static/js/quality-chooser.js [hunk 1]
```diff
@@ -44,7 +44,22 @@ $(document).ready(function() {
                   '&allowed=' + selectedAllowed +
                   '&preferred=' + selectedPreffered
         $.getJSON(url, function(data) {
-            $('#backlogged_episodes').text('Currently you have ' + data[1] + ' backlogged episodes. With this change you would have ' + data[0] + ' backlogged episodes');
+            var new_backlogged = parseInt(data[0])
+            var current_backlogged = parseInt(data[1])
+            var variation = Math.abs(new_backlogged - current_backlogged)
```
Comment (OmgImAlexis) [inline static/js/quality-chooser.js]: Also not sure why you're using Math.abs when it'll strip negative numbers, etc. just use `-`.

```
Math.abs('-1');     // 1
Math.abs(-2);       // 2
Math.abs(null);     // 0
Math.abs("string"); // NaN
Math.abs();         // NaN
```

File: static/js/quality-chooser.js [hunk 2]
```diff
@@ -44,7 +44,22 @@ $(document).ready(function() {
                   '&allowed=' + selectedAllowed +
                   '&preferred=' + selectedPreffered
         $.getJSON(url, function(data) {
-            $('#backlogged_episodes').text('Currently you have ' + data[1] + ' backlogged episodes. With this change you would have ' + data[0] + ' backlogged episodes');
+            var new_backlogged = parseInt(data[0])
```
Comment (OmgImAlexis) [inline static/js/quality-chooser.js]: Change this, the data being passed back should already be an int if it's not then you need to fix the python.

After that it's fine

File: static/js/quality-chooser.js (hunk 1 above)
Comment (fernandog) [inline static/js/quality-chooser.js]: @OmgImAlexis im using it to just calculate the variation. So i just display the variation number

If is greater or not i check in the IF.
Anyway i would need to use ABS to display a positive number
"increased 5"
"decreased 5"

Comment (OmgImAlexis) [inline static/js/quality-chooser.js]: All good, missed seeing the decrease listed below.

**Pull request merged**